test/noncomm_ring.lean

```diff
@@ -44,3 +44,7 @@ example : a ⚬ b = b ⚬ a := by noncomm_ring
 example : a ⚬ (b + c) = a ⚬ b + a ⚬ c := by noncomm_ring
 example : (a + b) ⚬ c = a ⚬ c + b ⚬ c := by noncomm_ring
 example : (a ⚬ b) ⚬ (a ⚬ a) = a ⚬ (b ⚬ (a ⚬ a)) := by noncomm_ring
+
```

Comment:
These are totally unrelated changes. I just had them lying around after https://github.com/leanprover-community/mathlib/pull/2858 and thought they would nicely complete the set of examples I provided there.

Please feel free to tell me to drop them if this seems silly, and I will do so.